### PR TITLE
Profile page, binding user to navbar info, theme and notification options, and snackbar

### LINKED ISSUE #34: As a user, I would like to see a profile page and be able to edit my info

### PR DESCRIPTION
**Pull request by esteeroramy**
Fixes #34 
image
image
image

### CODE REVIEW

**Comment by esteeroramy** [comment]
Confirm text on button follows theme

File: UI/js/login.js [hunk 1]
```diff
@@ -1,3 +1,21 @@
...
+(at your option) any later version.
+
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License
+along with this program.  If not, see <http://www.gnu.org/licenses/>.
+*/
```
Comment (ahmedqarmout) [inline UI/js/login.js]: Reminder to Update error codes

File: Templates/profile.pug [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+                .card
+                    .card-content.padding-0
+                        img.full-width(src=`/profilePicture/${user.picture}`)
+                .card
+                    .card-action
+                        .row
+                            .input-field.s12
+                                select#theme(name='theme')
+                                    each theme in themes
+                                        option(value=theme selected=(theme===user.theme) ? true : false) #{theme}
```
Comment (ahmedqarmout) [inline Templates/profile.pug]: I didn't see this XD

File: UI/js/profile.js [hunk 3]
```diff
@@ -0,0 +1,142 @@
...
+
+/**
+* Init function
+*/
+$(function () {
+    $('select').material_select();
+});
+
+$("#theme").on('change', function() {
+    $('bodyclass').removeClass();
```
Comment (ahmedqarmout) [inline UI/js/profile.js]: "->'
Var -> const for function definitions.
Define element as const at the beginning

File: Templates/profile.pug (hunk 2 above)
Comment (esteeroramy) [reply Templates/profile.pug]: shhh ;)

File: UI/js/profile.js (hunk 3 above)
Comment (esteeroramy) [reply UI/js/profile.js]: done

**Pull request merged**